recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
   git_url: "https://github.com/onnx/onnx.git"
   git_tag: {{ git_tag }}
   patches:
-{% if protobuf=="3.11.2" %}  # open-ce 1.2
+{% if protobuf=="3.14.*" %}  # open-ce 1.2
```

Comment:
I think it would be better to check if it is greater than 3.9, or if we can't do that, check if not equal to 3.9. That way it won't break the 1.2 builds.

Reply:
yeah - I'm trying that, but converting to float wont work because of the x.y.z. Ansible has a `compare_version` filter, but base jinja (and conda) does not.
Using `.split('.')` and converting each to ints to compare would work until you have the `*`  which doesn't covert. Still trying a few things.

Reply:
ok. I added something that's not pretty but works.